crates/ruff_python_formatter/tests/snapshots/format@statement__with.py.snap

```diff
@@ -678,6 +688,121 @@ with open(
     "/etc/hosts"  # This is an incredibly long comment that has been replaced for sanitization
 ):
     pass
+
+with aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa + bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb:
+    pass
+
+with aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa + bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb as b:
+    pass
+
+if True:
+    with anyio.CancelScope(
+        shield=True
+    ) if get_running_loop() else contextlib.nullcontext() as b:
+        pass
+```
+
+
+#### Preview changes
+```diff
+--- Stable
++++ Preview
+@@ -45,7 +45,9 @@
+ with a:  # should remove brackets
+     pass
+ 
+-with aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa + bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb as c:
++with (
++    aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa + bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
++) as c:
```

Comment:
This change might be controversial. I would say it's not ideal but the best we can do with the limitations given by Python 3.8, other than not parenthesizing and exceeding the configured line width.